algorithms-miscellaneous-5/src/test/java/com/baeldung/algorithms/subarray/maximum/BruteForceAlgorithmUnitTest.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.algorithms.subarray.maximum;
+
+import org.junit.jupiter.api.Assertions;
```

Comment:
Let's remove this unused import and also general comment about the tests is that the formatting is slightly off.